test/ramen-config/deploy

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+cd $(dirname $0)
+
+echo "Waiting until ramen deploymnet is ready"
+kubectl rollout status deploy/ramen-hub-operator \
+    --namespace ramen-system \
+    --timeout 60s \
+    --context hub
+
+echo "Creating s3 secret in ramen hub namespace"
+kubectl apply -f s3-secret.yaml --context hub
+
+echo "Updating ramen config map"
+# XXX Depends on minikube, will not work for external clusters.
+minio_dr1=$(minikube --profile dr1 -n minio service --url minio)
+minio_dr2=$(minikube --profile dr2 -n minio service --url minio)
```

Comment:
Need to replace minikube with kubectl command - there is an example in https://github.com/RamenDR/ramen/issues/743